### PR TITLE
Update NLS translations

### PR DESCRIPTION
**Pull request by pshipton**

### CODE REVIEW

**Comment by pshipton** [comment]
Fixing the line endings.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: runtime/nls/shrc/j9shr_ja.nls [hunk 1]
```diff
@@ -6911,7 +6911,7 @@ J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.system_action=
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
 
-J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=\u30aa\u30da\u30ec\u30fc\u30c6\u30a3\u30f3\u30b0\u30fb\u30b7\u30b9\u30c6\u30e0\u306e\u30da\u30fc\u30b8\u30fb\u30b5\u30a4\u30ba (%1$zu) \u306f\u3001\u30ec\u30a4\u30e4 %3$d \u30ad\u30e3\u30c3\u30b7\u30e5\u304c\u4f5c\u6210\u3055\u308c\u305f\u30b7\u30b9\u30c6\u30e0\u306e\u30da\u30fc\u30b8\u30fb\u30b5\u30a4\u30ba (%2$zu) \u3068\u7570\u306a\u308a\u307e\u3059\u3002
+J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=\u30aa\u30da\u30ec\u30fc\u30c6\u30a3\u30f3\u30b0\u30fb\u30b7\u30b9\u30c6\u30e0\u306e\u30da\u30fc\u30b8\u30fb\u30b5\u30a4\u30ba%1$zu \u306f\u3001\u30ec\u30a4\u30e4%3$d\u30ad\u30e3\u30c3\u30b7\u30e5\u304c\u4f5c\u6210\u3055\u308c\u305f\u30b7\u30b9\u30c6\u30e0\u306e\u30da\u30fc\u30b8\u30fb\u30b5\u30a4\u30ba%2$zu \u3068\u7570\u306a\u308a\u307e\u3059\u3002
```
[the remark this answers is not in the document]
Comment (pshipton) [reply runtime/nls/shrc/j9shr_ja.nls:6914]: How do you know? Google translate offers the following translation:
The operating system page size, %1$zu, differs from the page size, %2$zu, of the system on which the layer %3$d cache was created.

File: runtime/nls/shrc/j9shr_zh.nls [hunk 2]
```diff
@@ -6909,7 +6909,7 @@ J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.system_action=
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
 
-J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=\u64cd\u4f5c\u7cfb\u7edf\u7684\u9875\u9762\u5927\u5c0f\uff08%1$zu\uff09\u4e0e\u521b\u5efa\u5c42 %3$d \u7f13\u5b58\u7684\u7cfb\u7edf\u7684\u9875\u9762\u5927\u5c0f\uff08%2$zu\uff09\u4e0d\u540c\u3002
+J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=\u64cd\u4f5c\u7cfb\u7edf\u7684\u9875\u9762\u5927\u5c0f (%1$zu) \u4e0e\u521b\u5efa\u5c42%3$d\u7f13\u5b58\u7684\u7cfb\u7edf\u7684\u9875\u9762\u5927\u5c0f (%2$zu) \u4e0d\u540c\u3002
```
[the remark this answers is not in the document]
Comment (pshipton) [reply runtime/nls/shrc/j9shr_zh.nls:6912]: I don't know. Google translate gives the following, which seem fine.
This also applies to the next comment "Spaces around %3$d?", zh and zh_CN are the same translation.
The operating system's page size (%1$zu) is different from the system's page size (%2$zu) on which the tier %3$d cache was created.

File: runtime/nls/shrc/j9shr_de.nls [hunk 3]
```diff
@@ -6909,7 +6909,7 @@ J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.system_action=
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
 
-J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=Die Seitengr\u00f6\u00dfe (%1$zu) des Betriebssystems unterscheidet sich von der Seitengr\u00f6\u00dfe (%2$zu) des Systems, auf dem der Cache der Schicht %3$d erstellt wird.
+J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=Die Seitengr\u00f6\u00dfe%1$zu) des Betriebssystems unterscheidet sich von der Seitengr\u00f6\u00dfe%2$zu) des Systems, auf dem der Cache der Schicht %3$d erstellt wird.
```
[the remark this answers is not in the document]
Comment (pshipton) [reply runtime/nls/shrc/j9shr_de.nls]: Updated.

File: runtime/nls/shrc/j9shr_ko.nls [hunk 4]
```diff
@@ -6909,7 +6909,7 @@ J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.system_action=
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
 
-J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=\uc6b4\uc601 \uccb4\uc81c\uc758 \ud398\uc774\uc9c0 \ud06c\uae30(%1$zu)\uac00 \ub808\uc774\uc5b4 %3$d \uce90\uc2dc\uac00 \uc0dd\uc131\ub418\ub294 \uc2dc\uc2a4\ud15c\uc758 \ud398\uc774\uc9c0 \ud06c\uae30(%2$zu)\uc640 \ub2e4\ub985\ub2c8\ub2e4.
+J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=\uc6b4\uc601 \uccb4\uc81c\uc758 \ud398\uc774\uc9c0 \ud06c\uae30(%1$zu)\uac00 %3$d \uce90\uc2dc \uacc4\uce35\uc774 \ub9cc\ub4e4\uc5b4\uc9c0\ub294 \uc2dc\uc2a4\ud15c\uc758 \ud398\uc774\uc9c0 \ud06c\uae30%2$zu\uc640 \ub2e4\ub985\ub2c8\ub2e4.
```
[the remark this answers is not in the document]
Comment (pshipton) [reply runtime/nls/shrc/j9shr_ko.nls]: Updated.

File: runtime/nls/shrc/j9shr_ja.nls (hunk 1 above)
[the remark this answers is not in the document]
Comment (keithc-ca) [reply runtime/nls/shrc/j9shr_ja.nls:6914]: Fair enough.

File: runtime/nls/shrc/j9shr_zh.nls (hunk 2 above)
[the remark this answers is not in the document]
Comment (keithc-ca) [reply runtime/nls/shrc/j9shr_zh.nls:6912]: I don't really know either, but, as you say, Google seems to think it's fine.

File: runtime/nls/shrc/j9shr_zh_CN.nls [hunk 5]
```diff
@@ -6909,7 +6909,7 @@ J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.system_action=
 J9NLS_SHRC_OSCACHE_MMAP_STARTUP_JITSERVER_LAYER_FILE_DELETE_ERROR.user_response=Contact your service representative.
 # END NON-TRANSLATABLE
 
-J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=\u64cd\u4f5c\u7cfb\u7edf\u7684\u9875\u9762\u5927\u5c0f\uff08%1$zu\uff09\u4e0e\u521b\u5efa\u5c42 %3$d \u7f13\u5b58\u7684\u7cfb\u7edf\u7684\u9875\u9762\u5927\u5c0f\uff08%2$zu\uff09\u4e0d\u540c\u3002
+J9NLS_CC_OSPAGE_SIZE_MISMATCH_V1=\u64cd\u4f5c\u7cfb\u7edf\u7684\u9875\u9762\u5927\u5c0f (%1$zu) \u4e0e\u521b\u5efa\u5c42%3$d\u7f13\u5b58\u7684\u7cfb\u7edf\u7684\u9875\u9762\u5927\u5c0f (%2$zu) \u4e0d\u540c\u3002
```
[the remark this answers is not in the document]
Comment (keithc-ca) [reply runtime/nls/shrc/j9shr_zh_CN.nls:6912]: Google translate suggests this is ok as well.

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
Jenkins compile amac jdk21

**Comment by pshipton** [comment]
I can squash before this is merged.

**Comment by keithc-ca** [comment]
The [build](https://openj9-jenkins.osuosl.org/job/Build_JDK21_aarch64_mac_Personal/243) passed: I'll wait for you to squash or tell me you've decided against doing so.

**Comment by pshipton** [comment]
Squashed.

**Pull request merged**